package.json

```diff
@@ -1,19 +1,54 @@
 {
   "name": "tree-sitter-crystal",
   "version": "0.0.1",
-  "description": "",
+  "description": "Crystal grammar for tree-sitter",
+  "repository": "https://github.com/crystal-lang-tools/tree-sitter-crystal",
+  "license": "MIT",
+  "author": {
+    "name": "Gabriel Holodak",
+    "email": "gthepiper@gmail.com"
+  },
+  "contributors": [
+    {
+      "name": "Devonte",
```

Comment:
@devnote-dev Does this work for you?

Reply:
You can put my actual name if you want 😅 it's Devaune Whittle.